common/src/main/java/org/dash/wallet/common/ui/enter_amount/AmountView.kt

```diff
@@ -147,8 +154,8 @@ class AmountView(context: Context, attrs: AttributeSet) : ConstraintLayout(conte
     }
 
     private fun updateDashSymbols() {
-        binding.inputCurrencyToggle.isVisible = showCurrencySelector
-        binding.resultCurrencyToggle.isVisible = showCurrencySelector
+        binding.inputCurrencyToggle.isVisible = showCurrencySelector && !dashToFiat
```

Comment:
@ClaudeHangui yes, because `updateDashSymbols` can be called independently from `showCurrencySelector`. Those are dynamic properties that can be changed after `AmountView` initialization, and we can get both toggles hidden or both visible.
And also in general, if we assign visibility to toggles, I feel it should be done in the same way everywhere just to keep the intention clear and avoid confusion for anyone who looks at it in the future.